deploy/task-cad-checks.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cad-checks
+  namespace: configuration-anomaly-detection
+spec:
+  params:
+    - name: event
+      type: string
+      description: Json string of the event data
+  results:
+    - name: configuration-anomaly
+      description: Cloud infrastructure anomaly type (f.e. None, HostsDown, ...)
+    - name: customer-caused
+      description: Configuration anomaly caused by customer detected
+  steps:
+    - name: check-infrastructure
+      image: quay.io/app-sre/cad-cli:latest
```

Comment:
this image needs to be promoted before the PR is merged (or at least to be noted that it is required